### ISSUE TITLE
Guidance for proposals around new global constructors

### ISSUE THREAD

**Issue by ptomato** [tc39/how-we-work#93]
As per the May 2021 plenary (https://github.com/tc39/notes/blob/master/meetings/2021-05/may-26.md#discussion-of-globals-and-guidance-for-future-proposals), this repo should contain a document providing guidance to future proposals that want to introduce new global constructors / namespaces.

**Comment by ptomato** [comment]
One thing not mentioned in the plenary which could go in the guidance, is whether namespace objects should have a [@@toStringTag] property. Seems more helpful than harmful? See https://github.com/tc39/proposal-temporal/issues/1539#issuecomment-864194394

**Comment by bakkot** [comment]
> whether namespace objects should have a [@@toStringTag] property

Yes, [per the July 2020 meeting](https://github.com/tc39/notes/blob/5a3580a5a057f3160338025acc2b674e03cc00a3/meetings/2020-07/july-20.md#adding-reflectsymboltostringtag-2057) (see https://github.com/tc39/ecma262/pull/2057).

**Comment by ljharb** [comment]
The more specific question is, is that just top-level ones, or all? iow, `Temporal` surely must, per that consensus - but what about `Temporal.now`?
